### PR TITLE
Homework 1

### PR DESCRIPTION
**Pull request by solomencev**

### CODE REVIEW

File: src/test/java/com/epam/tc/hw1/DivTest.java [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+public class DivTest  extends AbstractTest {
+
+    @Test
+    public void divTest() {
+        double res = calculator.div(4, 2);
+        assertThat(res).isEqualTo(2);
+    }
```
Comment (obitkin) [inline src/test/java/com/epam/tc/hw1/DivTest.java]: Use dataproviders

_6.	All test data should be provided to test through the DataProviders_

File: src/test/java/com/epam/tc/hw1/MultTest.java [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+public class MultTest extends AbstractTest {
+
+    @Test
+    public void multTest() {
+        double res = calculator.mult(3, 2);
+        assertThat(res).isEqualTo(6);
+    }
```
Comment (obitkin) [inline src/test/java/com/epam/tc/hw1/MultTest.java]: Use dataproviders

_6.	All test data should be provided to test through the DataProviders_

File: src/test/java/com/epam/tc/hw1/SubTest.java [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+import org.testng.annotations.Test;
+
+
+public class SubTest extends AbstractTest {
+
+    @Test
+    public void subTest() {
+        double res = calculator.sub(7, 2);
+        assertThat(res).isEqualTo(5);
+    }
```
Comment (obitkin) [inline src/test/java/com/epam/tc/hw1/SubTest.java:15]: Use dataproviders

_6.	All test data should be provided to test through the DataProviders_

File: src/test/java/com/epam/tc/hw1/SumTest.java [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+public class SumTest extends AbstractTest {
+
+    @Test
+    public void sumTest() {
+        double res = calculator.sum(1, 2);
+        assertThat(res).isEqualTo(3);
+    }
```
Comment (obitkin) [inline src/test/java/com/epam/tc/hw1/SumTest.java]: Use dataproviders

_6.	All test data should be provided to test through the DataProviders_

File: src/test/java/com/epam/tc/hw1/DivTest.java [hunk 5]
```diff
@@ -0,0 +1,29 @@
+package com.epam.tc.hw1;
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+public class DivTest  extends AbstractTest {
```
Comment (obitkin) [inline src/test/java/com/epam/tc/hw1/DivTest.java]: `DivTest extends` 1 space sym

File: src/test/java/com/epam/tc/hw1/MultTest.java [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+
+    @DataProvider
+    public static  Object[][] multCorrectData() {
+        return new Object[][] {
```
Comment (obitkin) [inline src/test/java/com/epam/tc/hw1/MultTest.java]: `static Object` 1 space sym

**Review by obitkin — CHANGES_REQUESTED** [review]
Some small fixes are required

File: src/test/java/com/epam/tc/hw1/DivTest.java (hunk 1 above)
Comment (solomencev) [reply src/test/java/com/epam/tc/hw1/DivTest.java]: I removed simply tests and expanded the data set in the existing ones through the DataProviders

File: src/test/java/com/epam/tc/hw1/SubTest.java (hunk 3 above)
Comment (solomencev) [reply src/test/java/com/epam/tc/hw1/SubTest.java:15]: I removed simply tests and expanded the data set in the existing ones through the DataProviders

File: src/test/java/com/epam/tc/hw1/DivTest.java (hunk 5 above)
Comment (solomencev) [reply src/test/java/com/epam/tc/hw1/DivTest.java]: I removed space

File: src/test/java/com/epam/tc/hw1/MultTest.java (hunk 6 above)
Comment (solomencev) [reply src/test/java/com/epam/tc/hw1/MultTest.java]: I removed space

File: src/test/java/com/epam/tc/hw1/SumTest.java (hunk 4 above)
Comment (solomencev) [reply src/test/java/com/epam/tc/hw1/SumTest.java]: I removed simply tests and expanded the data set in the existing ones through the DataProviders

**Review by obitkin — CHANGES_REQUESTED** [review]
I just found that you don't have a pom file in your PR. So I can't check it. Also, you have so many defaults profiles. Which one do you run by default?

_by default should work profile which runs add and subtract operations tests_
```
<profiles>
        <profile>
            <id>SumSub</id>
            <activation>
                <activeByDefault>true</activeByDefault>
            </activation>
            <properties>
                <suite.xml.file>sumsub.xml</suite.xml.file>
            </properties>
        </profile>
        <profile>
            <id>MultDiv</id>
            <activation>
                <activeByDefault>true</activeByDefault>
            </activation>
            <properties>
                <suite.xml.file>multdiv.xml</suite.xml.file>
            </properties>
        </profile>
        <profile>
            <id>AllTest</id>
            <activation>
                <activeByDefault>true</activeByDefault>
            </activation>
            <properties>
                <suite.xml.file>alltests.xml</suite.xml.file>
            </properties>
        </profile>
    </profiles>
```

**Review by obitkin — APPROVED** [review]
Approved,

**Pull request merged**